raiden/tests/utils/eth_node.py

```diff
@@ -426,10 +388,6 @@ def _validate_jsonrpc_result(result):
 
     os.makedirs(logdir, exist_ok=True)
 
-    password_path = os.path.join(base_datadir, "pw")
-    with open(password_path, "w") as handler:
-        handler.write(DEFAULT_PASSPHRASE)
```

Comment:
I moved this closer to where the configuration is set, the idea is that it should be easier to understand the flow of the data.